server/user_test.go

```diff
@@ -38,3 +43,101 @@ func TestEncryptUserData(t *testing.T) {
 	require.NoError(t, err)
 	require.EqualValues(t, &expected, decrypted)
 }
+
+func TestStoreUserInfo(t *testing.T) {
+	tests := []struct {
+		name           string
+		kvSetUserErr   error
+		kvSetRemoteErr error
+		expectedErr    string
+	}{
+		{
+			name:           "No Error",
+			kvSetUserErr:   nil,
+			kvSetRemoteErr: nil,
+			expectedErr:    "",
+		},
+		{
+			name:           "Error Saving UserID",
+			kvSetUserErr:   &model.AppError{Message: "some error occurred while saving the user id"},
+			kvSetRemoteErr: nil,
+			expectedErr:    "some error occurred while saving the user id",
+		},
+		{
+			name:           "Error Saving RemoteID",
+			kvSetUserErr:   nil,
+			kvSetRemoteErr: &model.AppError{Message: "some error occurred while saving the remote id"},
+			expectedErr:    "some error occurred while saving the remote id",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			api := &plugintest.API{}
+			p := &Plugin{
+				MattermostPlugin: plugin.MattermostPlugin{
+					API: api,
+				},
+			}
+			p.setConfiguration(&configuration{
+				EncryptionKey: "demo_encrypt_key",
+			})
+
+			dummyInfo := &UserInfo{
+				UserID:   "dummyUserID",
+				RemoteID: "dummyRemoteID",
+			}
+
+			api.On("KVSet", "token_"+dummyInfo.UserID, mock.Anything).Return(tt.kvSetUserErr)
+			api.On("KVSet", "tbyrid_"+dummyInfo.RemoteID, mock.Anything).Return(tt.kvSetRemoteErr)
```

Comment:
what is the meaning of tbyrid_

Reply:
It is a constant in the code 
`tokenKeyByRemoteID = "tbyrid_"`